infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/util/PodEvents.java

```diff
@@ -30,6 +31,9 @@ private PodEvents() {}
    * Converts the time of {@link PodEvent} e.g. '2018-05-15T16:17:54Z' to the {@link Date} format
    */
   public static Date convertEventTimestampToDate(String timestamp) throws ParseException {
+    if (StringUtils.isBlank(timestamp)) {
```

Comment:
can you use 
org.eclipse.che.ide.util.StringUtils.isNullOrEmpty or
com.google.common.base.Strings.isNullOrEmpty ?

Reply:
yeah, sure

Reply:
+1